corpus/constants.txt

```diff
@@ -1,6 +1,6 @@
-==========
+================================================================================
```

Comment:
I ran `tree-sitter test --update`, so the diff is a bit ugly from the formatting changes that applies

Reply:
You know, I had wondered where these were coming from 😆